src/io/flutter/utils/JsonUtils.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.utils;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonNull;
+import com.google.gson.JsonObject;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class JsonUtils {
+
+  private JsonUtils() {
+  }
+
+  @Nullable
+  public static String getStringMember(@NotNull JsonObject json, @NotNull String memberName) {
+    if (!json.has(memberName)) return null;
+
+    final JsonElement value = json.get(memberName);
+    return value instanceof JsonNull ? null : value.getAsString();
+  }
+
+
```

Comment:
run code formatter on this file. Seems like there is some extra whitespace between methods.

Reply:
Good eyes! 👍